lib/dialects/mssql/query.js

```diff
@@ -67,16 +67,23 @@ class Query extends AbstractQuery {
           } else {
             resolve(this.formatResults());
           }
-        });
+        }, this.options.transaction.name);
+      } else if (_.includes(this.sql, 'SAVE TRANSACTION')) {
+        connection.saveTransaction(err => {
+          if (!!err) {
```

Comment:
Feel free to do so. The desired coding style is defined in eslintrc.json, it's just that the codebase does not fully comply with it yet.